docs/destinations/forwarding-events/snowbridge/concepts/scaling/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Scaling"
+date: "2023-02-27"
+sidebar_position: 150
+---
+
+# Scaling
+
+## Horizontal scaling
+
+Snowbridge is built to suit a horizontal scaling model, and you can safely deploy multiple instances of Snowbridge to consume the same input out-of-the-box (the kinesis source requires Dynamo DB tables to be created for this purpose). No addditional configuration or setup is required for the app to smoothly run across multiple instance/environment, compared to a single instance/environment.
+
+How to configure scaling behaviour will depend on the infrastructure you're using, and the use case you have implemented. For example, when scaling based on CPU usage, this metric will be affected by the size and shape of the data, which transformations and filters are used, and if using script transformations, the content of those scripts. 
+
+We recommend that you instrument your use case, and follow the best practices for the deployment infrastrucutre you're using to find a the scaling configuration which suits your requirements. Note that occasionally releases will improve efficiency, which in the past has had a large impact on metrics which can be used for scaling. So, it is advisable to monitor those metrics after any version update or transformation configuration change, to ensure that scaling behaves as expected.
+
+Additionally within each instance/environment, concurrency may be managed via the `concurrent_writes` setting - which provides a degree of control over throughput and resource usage. Snowbridge should consume as much data as possible, as fast as possible - a backlog of data or spike in traffic should cause the app's CPU usage to increase significantly. If spikes/backlogs do not induce this behaviour, and there are no target retries or failures (see below), then the `concurrent_writes` can be increased to induce this behaviour.
```

Comment:
```suggestion
Additionally within each instance/environment, you can manage concurrency via the `concurrent_writes` setting, which provides a degree of control over throughput and resource usage. Snowbridge should consume as much data as possible, as fast as possible — a backlog of data or spike in traffic should cause the app’s CPU usage to increase significantly. If spikes/backlogs do not induce this behaviour, and there are no target retries or failures (see below), then you can increase the `concurrent_writes`.
```

Reply:
Shoud we move this to the Concurrency section? Seems more natural to mention after the explanation of goroutines etc.

Reply:
I think that too - I landed on putting this bit here because it's a piece of info that you'll specifically want if you have volume, but the app isn't processing enough data to induce the scaling you need. The thinking is that in such a scenario you might read the scaling docs to see what you're doing wrong, but not think that the concurrency section has the piece of info you actually need.

So, the way I see it is there's a bit in the scaling section saying "you can increase concurrency to eat more resources and induce scaling", then there's a section explaining what concurrency is doing.

I'm easy either way - just explaining the thought process. If you still think it's better below, I'll move it - just let me know. :)